tests/compile-fail/data_race/release_seq_race.rs

```diff
@@ -40,9 +40,9 @@ pub fn main() {
         });
 
         let j3 = spawn(move || {
-            sleep(Duration::from_millis(1000));
+            sleep(Duration::from_millis(5000));
```

Comment:
5 seconds seems rather excessive. I'd like to avoid having tests run too long. Is there something taht can be dine to avoid this?

Reply:
The change was from when I was trying to debug the CI failure that I forgot to revert, i have rolled back to more reasonable values.